### PR TITLE
fix: use utc on echart tooltips

### PR DESCRIPTION
**Pull request by rephus**
Closes: #2257

### Description:

https://echarts.apache.org/en/option.html#useUTC

image

### CODE REVIEW

**Review by owlas — COMMENTED** [review]
@rephus does the backend always return UTC? 

How does this setting change things if the backend returns:
* A raw UTC timestamp
* A timestamp with a timezone
* A truncated day/week/month/year
?

**Comment by rephus** [comment]
afaik the backend always returns UTC

customer.created returns `raw` timestamp,
With useUTC:true 

image
image

By month
image


With useUTC:false (wrong ) 

image

**Comment by owlas** [comment]
I guess the backend will return whatever the user queries from their database, which could be in UTC or not. I like the idea of showing everything in UTC initially to we can be sure we've normalised everything, then we can later expose a config to alter that.

Is UTC also shown in all other places?
* big number
* explore table 
* table viz
* chart axes
* filters?

Especially the filters one, we'd need to make sure the same timezone is being sent in the query too

**Comment by rephus** [comment]
The filters were already using UTC, on a fix I made recently https://github.com/lightdash/lightdash/pull/2180/files

**Comment by rephus** [comment]
> I guess the backend will return whatever the user queries from their database, which could be in UTC or not. I like the idea of showing everything in UTC initially to we can be sure we've normalised everything, then we can later expose a config to alter that.
> 
> Is UTC also shown in all other places?
> 
> * big number
> * explore table
> * table viz
> * chart axes
> * filters?
> 
> Especially the filters one, we'd need to make sure the same timezone is being sent in the query too

Table 
image

Big number

image


In any case, I don't think any of those charts are related with the fix, or the issue https://github.com/lightdash/lightdash/issues/2257

File: packages/frontend/src/hooks/echarts/useEcharts.ts [hunk 1]
```diff
@@ -597,6 +597,7 @@ const useEcharts = () => {
     return {
         xAxis: axis.xAxis,
         yAxis: axis.yAxis,
+        useUTC: true,
```
Comment (nathaliabj) [inline packages/frontend/src/hooks/echarts/useEcharts.ts:600]: 🥳

**Pull request merged**